pkg/status/templates/otlp.tmpl

```diff
@@ -0,0 +1,10 @@
+{{/*
+NOTE: Changes made to this template should be reflected on the following templates, if applicable:
+* cmd/agent/gui/views/templates/generalStatus.tmpl
+*/}}
+====
+OTLP
+====
+  Status: {{ if .otlpStatus }}Enabled{{else}}Not enabled{{ end }}
```

Comment:
Is there a case when 'Not enabled' can be displayed ? As you render this only when `otlp.IsEnabled(config.Datadog)` is true (in [pkg/status/render.go](https://github.com/DataDog/datadog-agent/pull/11822/files#diff-7ccf15f4120b36c61d49a33a3110c87f1d559eeeaf540ab047722d66734a1d2fR97))

Reply:
Good catch! It looks like other features do render when they are not enabled https://github.com/DataDog/datadog-agent/blob/e2cd7ee8be48c702a1d3483c0a6f97e062dc6a1a/pkg/status/templates/runtimesecurity.tmpl#L4-L6 so I would suggest doing the same here so that:
1. if OTLP is available but not enabled, we render a section that says 'Not enabled'
2. if OTLP is available and enabled, we render a section that says 'Enabled'
3. if OTLP is not available (e.g. in the serverless agent or any other that doesn't have the `otlp` build tag) we don't render

Reply:
Fixed with [1c2ab00](https://github.com/DataDog/datadog-agent/pull/11822/commits/1c2ab00810f5dda2e8c84cdb7d294e55f829a9f6)